File: SESION3/dart/alquiler.dart
class Alquiler
{
    int numeroJugadores= 0;
    double precioTotal= 0;
    String localizacion= "";

    Alquiler(int numeroJug, String local){
        numeroJugadores=numeroJug;
        localizacion=local;
    }

    double getPrecioTotal(){return precioTotal;}

    int getNumeroJugadores(){return numeroJugadores;}

    void incrementarCoste(double incremento){
        precioTotal+= incremento;
    }

    String getLocalizacion(){return localizacion;}

}

File: SESION3/C++/Alquiler.cpp
#include "Alquiler.h"

    Alquiler::Alquiler(int numeroJug, string local){
        numeroJugadores=numeroJug;
        localizacion=local;
    }

    void Alquiler::incrementarCoste(float incremento){
        precioTotal+=incremento;
    }